client-side/src/components/report.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import GenericButton from '../stories/Button/GenericButton';
+import DateInputs from "../stories/DateTime/DateInputs";
+const DateTimePicker = () => {
+    const [startDate, setStartDate] = useState('');
+    const [endDate, setEndDate] = useState('');
+    const storeArr = () => {
```

Comment:
this function do nothing, why?

Reply:
The function is not empty. where it stores the data in an array and sends the array to a function that displays the table according to the data.

Reply:
so why is fillData is commented?
also remove the alert.